src/Platform.c

```diff
@@ -204,12 +204,38 @@ void PltJoinThread(PLT_THREAD* thread) {
 #endif
 }
 
+void PltDetachThread(PLT_THREAD* thread)
+{
+    // Assume detached threads are no longer active
+    activeThreads--;
+
+#if defined(LC_WINDOWS)
+    // According MSDN:
+    // "Closing a thread handle does not terminate the associated thread or remove the thread object."
+    CloseHandle(thread->handle);
+#elif defined(__vita__)
+    sceKernelDeleteThread(thread->handle);
```

Comment:
It looks like this function terminates the thread, so it should probably be a no-op instead.

https://github.com/libsdl-org/SDL/commit/512355d7aa7ee267d31a43909bf2e2748f4ca957

Reply:
Ok, I'll fix this up to do something similar.